src/main/java/androidx/widget/TextView.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.widget
+
+import android.text.Editable
+import android.text.TextWatcher
+import android.widget.TextView
+
+/**
+ * Returns the text displayed by the [TextView].
+ */
+fun TextView.content() = text.toString()
+
+/**
+ * Returns `true` if this char sequence is empty (contains no characters).
+ */
+fun TextView.isEmpty() = text.isEmpty()
```

Comment:
I don't find these two or `clear()` that useful.

Reply:
I will remove.

Reply:
Fixed.

Reply:
Forget to push up changes?

Reply:
I removed both `isEmpty()` and `clear()`.